### ISSUE TITLE
UserMenu accessibility

### ISSUE THREAD

**Issue by JensSteckhan** [SAP/fundamental-ngx#2672]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
When I add menu items to the shellbar user menu, they should be reachable via keyboard.
Preferably via the up- or down arrow.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
18.0

#### If this is a bug, please provide steps for reproducing it.
Create a simple shellbar with a menu
html:
<fd-shellbar><fd-shellbar-actions [user]="userName" [userMenu]="userMenu" title={{userName.fullName}} [closePopoverOnSelect]="true">...
ts:
this.userMenu = [
        { text: 'menu item 1', callback: this.foo.bind(this) },
        { text: 'menu item 2', callback: this.bar.bind(this) }
      ]

Run your application hit tab untill the user button has the focus.
Hit return and the menu will appear. Now try to reach the menu items with your keyboard.
It is not possible.

#### Is there anything else we should know?
I find your examples at https://sap.github.io/fundamental-ngx helpful.
Often I find that your examples behave slightly differently when I copy the code.
https://sap.github.io/fundamental-ngx/#/core/shellbar shows user menus which do react on arrow up and down (though there is another bug: pressing the key down not only changes the focus of the menu item, but wrongfully scrolls the page at the same time).
No idea why the menu is selected in your examples.
